blog/models.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.conf import settings
+
+# Create your models here.
+class User(AbstractUser):
+    pass
+
+
+class Post(models.Model):
+    owner = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, related_name='posts')
+    title = models.CharField(max_length=200)
+    content = models.TextField()
+    created_time = models.DateTimeField(auto_now_add=True)
+    def __str__(self):
+        return self.title
+
+
+class Commentary(models.Model):
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, related_name='commentary')
+    post = models.ForeignKey(Post, on_delete=models.CASCADE, related_name='commentary')
+    created_time = models.DateTimeField(auto_now_add=True)
+    content = models.TextField()
+
+    def __str__(self):
+        return self.post.title
```

Comment:
For the Commentary model, the `__str__` method is returning the title of the post. It would be more appropriate to return a string that represents the commentary itself, perhaps the first few characters of the commentary content. For example: `return self.content[:50]`